pipeline/steps/run-sonarcloudanalysis.yaml

```diff
@@ -0,0 +1,23 @@
+parameters:
+  - name: command
+    type: string
+
+steps:
+- task: SonarSource.sonarcloud.14d9cde6-c1da-4d55-aa01-2965cd301255.SonarCloudPrepare@1
+  displayName: 'Prepare analysis on SonarCloud'
+  inputs:
+    SonarCloud: SonarCloud
+    organization: hmcts
+    scannerMode: Other
+
+- task: Gradle@2
```

Comment:
this task already exists https://github.com/hmcts/hmi-api-gateway-fh/blob/e0e3f99e78543f1636ca4ff5ba0361f77450cc6d/pipeline/steps/run-gradlew.yaml

please use existing modules, we try to break down to single tasks so we don't need to repeat the code

Reply:
Used existing Gradle tasks into sonarcloud.